.pipelines/testchecklist.md

```diff
@@ -3,207 +3,133 @@
-- Click on "Extend" -> The specified Extend Time is added to both "Max Time" and "Remaining Time".
-- Click on "Stop Session" -> The session is stopped and the panel returns to the previous state.
-- Start a session with a very short Max Time. Make sure Auto Extend is active.
-- While the session is running verify that:
-  - The session status, shown on the top of the session panel, says "Connected" and the time shown is the remaining time.
-  - The "Render" button shows the status icon and has the session information (status/remaining time) in the tooltip.
-  - From the session panel, "inspect session" is enabled.
-  - Click in "inspect session" -> ArrInspector starts and shows live data from the ARR session (you will see the logs).
-  - The panel behind the session panel (model selection panel) becomes active and allows you to select a model.
-- Wait for the session to be near expiration time -> The session time will automatically extend.
-
-### Model selection panel
-
-- Make sure the session is connected and the panel is enabled.
-- Make sure the "input mode" is set to "From storage container".
-- Verify that the blob explorer for the model selection is working (it behaves like the model input).
-- Select a model to load and click load -> A progress bar appears and will load the model.
-- Once it's loaded press "unload" on the top -> Return to the loading page.
-- Double click on a model -> It loads the model.
-- Select "From SAS URL" as input mode -> See a text box to enter the URL.
-- Try builtin://Engine -> The engine demo model is loaded
-- Unload, generate a Sas URI from Azure Storage Explorer to a converted asset (.arrAsset) and input it in the field -> The model is loaded
-- Unload the model
-- From the main settings, disconnect to the Storage Account, by changing for example the name, and pressing "Retry"
-- Check that the only available option now is "From Sas URL" and check if it still works.
+- Wait for the session to start
+- Open 'Sessions...' again
+- Check that the *Max Time* and *Remaining Time* are the ones you specified
+- Check that now the *Stop* button is enabled
+- Click on *Extend Now* -> The specified time is added to both *Max Time* and *Remaining Time*
+- Wait a bit (or use a session with a shorter max time) and check the log to see that the auto-extension kicks in
+- Click the *Open ArrInspector* button and check that it's working
+
+### Models / Rendering / Camera
+
+- With a running session, test the *Load Model from Storage* and *Load Model with URL* buttons
+- There should be progress bar at the bottom when any model loading is ongoing
+- Verify that the first loaded model gets centered in the viewport
+- You can add multiple models, use *Remove Models* to clear them all
+- Test that the *Model Scale* option works
+- Use the *Camera...* button to adjust camera options (near/far plane, camera speed)
+- In the settings, disconnect from the storage account (e.g. by the name to be invalid)
```

Comment:
What do you mean with e.g. by the name to be invalid?
By setting the name to some invalid name, or doing  something else to make the current name invalid?